registry/michaelwillis/dragonfly-reverb/3.2.7/package.yaml

```diff
@@ -0,0 +1,42 @@
+---
+name: Dragonfly Reverb
+creator: Michael Willis
+license: gpl-3.0
+screenshotUrl: https://central.owlplug.com/products/dragonflyreverb/dragonflyreverb.png
+description: Dragonfly Reverb is a bundle of free audio effect plugins for Linux,
+  MacOS, and Windows.
+pageUrl: https://michaelwillis.github.io/dragonfly-reverb/
+donateUrl: https://michaelwillis.github.io/dragonfly-reverb/donations.html
+type: effect
+tags:
+- Reverb
+bundles:
+- name: Win32 Plugins
+  targets:
+  - win32
+  format: various
+  downloadUrl: https://github.com/michaelwillis/dragonfly-reverb/releases/download/3.2.7/dragonfly-reverb-win32-3.2.7.zip
+  downloadSha256: 27cfd66fe026d0c98c1754560e4b44b78e8395ef45790c544e03f18f92747952
+  fileSize: 15000000
+- name: Win64 Plugins
+  targets:
+  - win64
+  format: various
+  downloadUrl: https://github.com/michaelwillis/dragonfly-reverb/releases/download/3.2.7/dragonfly-reverb-win64-3.2.7.zip
```

Comment:
Based on the AV report, the following file hosted on github is detected as potentially dangerous by some common vendors:
* **dragonfly-reverb-win64-3.2.7.zip**: https://www.virustotal.com/gui/file-analysis/MGVlODBkMzlkMDU3NGRiM2Q1MmQxMjNhZDk2Mjg1NTg6MTY3NDAzMTc3Nw==/detection

This PR should not be merged for now.

Reply:
False positive by common AV vendors resolved